docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -155,6 +155,10 @@ If the value is unspecified, there is no global limit on the number of active do
 and Envoy will emit a warning indicating this at startup. To disable the warning without setting a
 limit on the number of active downstream connections, the runtime value may be set to a very large
 limit (~2e9).
+Listeners (including the admin listener) can opt out of this global connection limit by setting
```

Comment:
Probably should clarify that for the admin listener the config is not in the `Listener` proto, and point to the correct one.